File: src/main/java/com/optica/services/search/FilterJobOfferService.java
package com.optica.services.search;

import java.util.List;
import java.util.Map;

import com.optica.domain.JobOffer;

public interface FilterJobOfferService {
	
	List<JobOffer> filterJobOffer(String field, String[] values);

	List<JobOffer> filterJobOffer(Map<String, String[]> filters);
	
}

File: src/main/kotlin/com/optica/services/search/FilterJobOfferService.kt
package com.optica.services.search

import com.optica.domain.JobOffer

interface FilterJobOfferService {

    fun filterJobOffer(field: String, values: Array<String>): List<JobOffer>

    fun filterJobOffer(filters: Map<String, Array<String>>): List<JobOffer>

}

